### DISCUSSION TITLE
learn/workflow/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#251] [Website Discussions]
# learn/workflow/

This page will be dedicated to professional Final Cut Pro Workflows.

https://fcp.cafe/learn/workflow/

**Comment by devbenegal** [comment]
Larry Jordan has an excellent article on RAM and Storage requirements:
https://larryjordan.com/articles/thoughts-on-configuring-m2-macs-for-video-production-and-editing/

The tldr is: 

**RAM**
-For HD editing, 16 GB of RAM is enough.
-For 4K editing, 32 GB of RAM is enough.
-For multicam editing, 64 GB of RAM is enough.
My recommendation is to get a system with 32 or 64 GB of RAM.

**STORAGE**
My recommendation is an editing system with 1 – 2 TB of internal storage
